maven/impl-maven-embedded/src/test/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/BuildOutputTestCase.java

```diff
@@ -67,7 +67,7 @@ private void verifyBuildContent(boolean shouldContain) {
     }
 
     private void assertBuildStdoutContains(String expectedString, boolean shouldContain){
-        StringBuffer assertWarn = new StringBuffer("The build output printed on stdout should ");
+        StringBuilder assertWarn = new StringBuilder("The build output printed on stdout should ");
```

Comment:
This has nothing to do with JDK version, and it might be intentional if accessed from multiple threads.

Reply:
I don't think there's any harm in using the StringBuilder. No other threads are accessing the same instance and StringBuilder should be faster, but in this case I don't mind having the StringBuffer back. Personally, I would keep the StringBuilder